backend/app_fastapi_v2.py

```
@@ -28,26 +48,92 @@ async def ping():
 @v2.post("/generate_review")
-async def api_generate_review(request: ReviewRequestV2):
+async def api_generate_review(request: ReviewRequestV2, current_user: dict = Depends(get_current_user)):
     try:
         review = generate_review(**request.model_dump())
         return {"review": review}
     except Exception as e:
         raise HTTPException(status_code=500, detail=str(e))
 
 @v2.post("/generate_self_review")
-async def api_generate_self_review(request: SelfReviewRequestV2):
+async def api_generate_self_review(request: SelfReviewRequestV2, current_user: dict = Depends(get_current_user)):
```

Ruff

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)